### DISCUSSION TITLE
CIPP Sazerac release through Github action errors

### DISCUSSION THREAD

**Discussion by ADVTECH-AU** [KelvinTegelaar/CIPP#1180] [Q&A] [answered]
Have been a CIPP Customer for several months but syncing the the Sazerac release through today it failed and i have not been able get my CIPP up and running again

Please help me figure out why.

![image](https://user-images.githubusercontent.com/109508407/199190751-4fa86f40-d77c-4dc3-925c-0d3253530eed.png)

**Comment by KelvinTegelaar** [comment, marked as answer]
Hosted clients can get basic email support, community support is available is Discord.
